package.json

```diff
@@ -60,11 +60,13 @@
     "@wordpress/block-editor": "^6.1.9",
     "@wordpress/blocks": "^9.1.4",
     "@wordpress/components": "^13.0.1",
-    "@wordpress/compose": "^3.23.0",
+    "@wordpress/compose": "^3.25.3",
+    "@wordpress/core-data": "^3.1.9",
     "@wordpress/data": "^5.1.3",
     "@wordpress/date": "^3.15.0",
     "@wordpress/element": "^2.19.0",
     "@wordpress/i18n": "^3.17.0",
+    "@wordpress/notices": "^3.1.3",
```

Comment:
Once the block is in its own package we can clean up a lot of these dependencies here!